### PR TITLE
Feature/skjemasteg

### PR DESCRIPTION
**Pull request by henninghaakonsen**
Laget felleskomponent for skjemasteg med barn(react children) som vanligvis vil være et skjema. 
Komponenten tar inn tittel (på steget), forrige og neste knapper (som kun er synlig om funksjonen sendes inn) og en boolean som forteller om skjemaet sendes inn.

### CODE REVIEW

File: src/frontend/komponenter/Fagsak/useFagsakApi.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+const useFagsakApi = (
+    settVisFeilmeldinger: (visFeilmeldinger: boolean) => void,
+    settFeilmelding: (feilmelding: string) => void
+) => {
+    const history = useHistory();
+    const [senderInn, settSenderInn] = useState(false);
+
+    const fagsakDispatcher = useFagsakDispatch();
+    const context = useOpprettBehandlingContext();
+
```
Comment (MengZhuNAV) [inline src/frontend/komponenter/Fagsak/useFagsakApi.ts]: The function useFagsakApi() is supposed to be reused by all pages, but the context and the opprettBehandling() code below seems to be specific to the OpprettBehandling page. Is this by intention?

Comment (henninghaakonsen) [reply src/frontend/komponenter/Fagsak/useFagsakApi.ts]: Yes, each "action" may use context. One could argue that it should be passed as a parameter to the function though 🤔

File: src/frontend/komponenter/Fagsak/Opprett/OpprettBehandling.tsx [hunk 2]
```diff
@@ -1,94 +1,46 @@
...
-    const history = useHistory();
     const context = useOpprettBehandlingContext();
-    const dispatch = useOpprettBehandlingDispatch();
+
     const [visFeilmeldinger, settVisFeilmeldinger] = React.useState(false);
     const [opprettelseFeilmelding, settOpprettelseFeilmelding] = React.useState('');
 
+    const { opprettBehandling, senderInn } = useFagsakApi(
+        settVisFeilmeldinger,
+        settOpprettelseFeilmelding
```
Comment (MengZhuNAV) [inline src/frontend/komponenter/Fagsak/Opprett/OpprettBehandling.tsx]: Maybe we can handle the exceptions and error messages in Skjemastag, so we don’t need to define these states for every page and the parameters to useFagsakApi can be simplified

**Comment by MengZhuNAV** [comment]
The design may need to be polished, but we can do it later when all pages are ready👍

File: src/frontend/komponenter/Fagsak/Opprett/OpprettBehandling.tsx (hunk 2 above)
Comment (henninghaakonsen) [reply src/frontend/komponenter/Fagsak/Opprett/OpprettBehandling.tsx]: The problem is that then we need to define props for the children and this is problematic. We could solve this with providers, but I dont think we need this complexity.

**Pull request merged**